pages/linux/nmcli.md

```diff
@@ -1,20 +1,17 @@
 # nmcli
 
 > A command-line tool for controlling NetworkManager.
-> More information: <https://man.archlinux.org/man/nmcli.1>.
+> This is a forwarding page to the advanced commands of `nmcli`.
+> More information: <https://manned.org/nmcli>.
 
-- Check the nmcli version:
+- Run an `nmcli` command:
```

Comment:
Is this too generic?

Reply:
nmcli doesnt do anything without the subcommand, I think its fine

Reply:
How about
```suggestion
- Run an `nmcli` subcommand:
```